pyoos/parsers/nerrs.py

```diff
@@ -116,7 +119,7 @@ def get_station(feature):
             s = Station()
             s.uid = feature['Station_Code']
             s.name = feature['Station_Name']
-            s.location = sPoint(float(feature['Longitude']), float(feature['Latitude']), 0)
+            s.location = sPoint(feature['Longitude'], feature['Latitude'], 0)
```

Comment:
It is safer, sure. But no longer necessary b/c the feature property lat & lon values are now being set as float type (see my commit on nerrs_soap.py), where previously it was left as the string from the original source metadata. So, safer, but redundant and unnecessary.